astropy/cosmology/flrw/base.py

```diff
@@ -1527,7 +1527,11 @@ def nonflat(self: _FlatFLRWMixinT) -> _FLRWT:
         return inst
 
     def clone(
-        self, *, meta: Mapping | None = None, to_nonflat: bool = None, **kwargs: Any
+        self,
```

Comment:
Tsk tsk, what's up with this?

Reply:
Apparently changing the `to_nonflat` type annotation from the incorrect `bool` to the correct `bool | None` causes the line to go over the 88 character limit.

Reply:
LGTM!